src/hooks/useSubscriptionPlan.ts

```diff
@@ -0,0 +1,32 @@
+import {useMemo} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import {isPolicyOwner} from '@libs/PolicyUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+
+function useSubscriptionPlan() {
+    const [policies] = useOnyx(ONYXKEYS.COLLECTION.POLICY);
+    const [session] = useOnyx(ONYXKEYS.SESSION);
+
+    // Filter workspaces in which user is the admin and the type is either corporate (control) or team (collect)
+    const adminPolicies = useMemo(
```

Comment:
```suggestion
    // Filter workspaces in which user is the owner and the type is either corporate (control) or team (collect)
    const ownerPolicies = useMemo(
```
(will need updates to the var name below too)

Reply:
Good point!